test/runtests.jl

```diff
@@ -158,6 +164,12 @@ end
         @test length(callsites) == 1
         ci = first(callsites).info
         @test isa(ci, Cthulhu.UncachedCallInfo)
+        if Cthulhu.EFFECTS_ENABLED
+            effects = Cthulhu.get_effects(ci)
+            @test !Core.Compiler.is_consistent(effects)
+            @test Core.Compiler.is_effect_free(effects)
+            @test !Core.Compiler.is_nothrow(effects)
```

Comment:
Maybe `@test !Core.Compiler.is_terminates(effects)` also? Mutually recursive function calls usually prohibit a proof of termination.